### ISSUE TITLE
Win 11, trying to upgrade from 2.1.18 to 2.2.49

### ISSUE THREAD

**Issue by chizy21** [kauffman12/EQLogParser#209]
Install is stuck at the installing screen.

After installing the microsoft installer.  I launched the 2.2.49 exe installer.

At Info screen click on next. then next agin that the path screen.  Then tells me that program/folder already exists.
I then click yes, then the additional task screen I click the box for create a desktop icon.
AT ready to install, I click the install button -- then it is stuck at the installing screen.  I have removed the old version and it gets stuck at the scame screen.  I can create a loom video if you need it.

**Comment by kauffman12** [comment]
I'd go into windows add/remove programs and uninstall anything EQLogParser related. It doesn't remove your user settings so it shouldn't hurt anything. Then run with debug from the command line:

EQLogParser-install-2.2.49.exe /LOG="install_log.txt"

Then maybe post the log or see if anything bad shows up.

**Comment by chizy21** [comment]
2024-10-27 16:40:24.643   Log opened. (Time zone: UTC-05:00)
2024-10-27 16:40:24.644   Setup version: Inno Setup version 6.2.2
2024-10-27 16:40:24.644   Original Setup EXE: F:\games\Everquest\eq\utils\EQLogParser-install-2.2.49.exe
2024-10-27 16:40:24.644   Setup command line: /SL5="$91124,16084656,789504,F:\games\Everquest\eq\utils\EQLogParser-install-2.2.49.exe" /SPAWNWND=$41260 /NOTIFYWND=$E1184 /LOG=install_log.txt
2024-10-27 16:40:24.645   Compatibility mode: Yes (Installer)
2024-10-27 16:40:24.645   Windows version: 10.0.22000  (NT platform: Yes)
2024-10-27 16:40:24.645   64-bit Windows: Yes
2024-10-27 16:40:24.662   Processor architecture: x64
2024-10-27 16:40:24.663   User privileges: Administrative
2024-10-27 16:40:24.669   Administrative install mode: Yes
2024-10-27 16:40:24.669   Install mode root key: HKEY_LOCAL_MACHINE
2024-10-27 16:40:24.669   64-bit install mode: Yes
2024-10-27 16:40:24.676   Created temporary directory: C:\Users\lsp00\AppData\Local\Temp\is-9A0VR.tmp
2024-10-27 16:40:31.069   Found 45 files to register with RestartManager.
2024-10-27 16:40:31.069   Calling RestartManager's RmGetList.
2024-10-27 16:40:31.080   RmGetList finished successfully.
2024-10-27 16:40:31.080   RestartManager found no applications using one of our files.

**Comment by kauffman12** [comment]
If you run 'dotnet --list-runtimes' is there a line like this? With some version of 8.0 installed.
Microsoft.WindowsDesktop.App 8.0.8 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

Besides checking that I'm not sure. Maybe disable anti-virus temporarily or just let it try to run longer. Something may be slowing things down but I don't know what. 

Also, your Windows 11 version looks like it's the original without any updates applied. You might want to update Windows but I don't see anything about Inno Setup failing on your version.

Lastly, I guess you could try looking in Windows Event Viewer. If you look for Event Viewer and go to Windows Log and Application. Then do a right-click clear logs to empty it out so it's easier to see. Then run the installer and see anythings shows up in that log related to EQLogParser.

**Comment by chizy21** [comment]
![image](https://github.com/user-attachments/assets/b96be078-f235-459f-b1df-a3d449151a58)

**Comment by kauffman12** [comment]
You could try doing:
cd %TEMP%

Then try making a file in that directory it takes you to. Make sure you have permissions to the temp directory and that it exists. Not sure why it wouldn't but that's the only other thing going on. It needs to create some temp files during the install.

**Comment by kauffman12** [comment]
In the mean time I can add a zip file with everything under Releases. You could take this and from the EQLogParser folder under Program Files and create a shortcut manually.

https://github.com/kauffman12/EQLogParser/raw/refs/heads/master/Release/EQLogParser-bin-2.2.49.zip

**Comment by TheVons** [comment]
FWIW I am on a fully patched install in Win 11 and having this same issue. Did DL the zip above and running portable.

**Comment by kauffman12** [comment]
Well next version I'll add some debug code that hopefully shows up in the install log. There is also a newer version of Inno Setup so maybe I'll try that.  I just don't see any major windows 11 issues that it fixes unless you're on Arm or ia64. Not normal intel/amd systems.

**Comment by chizy21** [comment]
Here is my hardware setup .....
[SOLACE-DT.txt](https://github.com/user-attachments/files/17670227/SOLACE-DT.txt)

**Comment by kauffman12** [comment]
Try the latest installer. 2.2.50.  If it fails again the try running with the LOG option and see what it says. There's more debug now.

**Comment by chizy21** [comment]
![image](https://github.com/user-attachments/assets/ad73135c-d060-4181-ad0d-3a0ad179bf21)

**Comment by kauffman12** [comment]
So, two things. You didn't type the full file name with the extension. And you need to bring up a command prompt. Don't run this from power shell. When you bring up the start menu type command or cmd and click on command prompt.  Then change to that Downloads directory.

EQLogParser-install-2.2.50.exe /LOG=debug.log

**Comment by chizy21** [comment]
[debug.log](https://github.com/user-attachments/files/17689100/debug.log)

**Comment by kauffman12** [comment]
So do you still have a really old version installed?  If you bring up Add or Remove Programs and type in EQLogParser. Does something besides like 2.2.49 show up?

**Comment by chizy21** [comment]
![image](https://github.com/user-attachments/assets/feee3c6f-9bcf-4903-b3c3-b6b5927df3b2)

**Comment by chizy21** [comment]
This is still happening even after uninstalling the old version, going to the appdata folder and deleting the EQLogParser folder there.[debug.log](https://github.com/user-attachments/files/17689240/debug.log)

**Comment by kauffman12** [comment]
Well it's failing to uninstall the old MSI version even though it's not installed anymore. You could try rebooting and see if that helps. But I may just remove that uninstall step and people will have to manually uninstall it going forward. There's no easy way to force the uninstall to work when something is broken about it.

In the mean time if you really wanted to you could try this microsoft app that tries to fix problems with apps being removed.

https://support.microsoft.com/en-us/topic/fix-problems-that-block-programs-from-being-installed-or-removed-cca7d1b6-65a9-3d98-426b-e9f927e1eb4d

Basically you go to that link and download troubleshooter. Run that. Click next. Select Uninstalling. Then see if EQLogParser shows up in the list. If it does then select it and click Next. I don't know what the next steps are but hopefully it can remove it for real.

**Comment by chizy21** [comment]
EQlog does not show up in the list and rebooting does not help.

**Comment by kauffman12** [comment]
Ok just try 2.2.51. It no longer tries to find and remove old MSI installs.

**Comment by chizy21** [comment]
That worked

**Comment by kauffman12** [comment]
Ok sounds good
